internal/incoming/odfi/testdata/recon.ach

```diff
@@ -0,0 +1,2 @@
+5220Robert Smith                        MOOVYYYYYYPPDaccount va2005262005271491323274270000085
```

Comment:
should there be a large recon file to use in a test?

Reply:
Yea, we could. There are a lot of tests around `ach.MergeFiles` in moov-io/ach.

Reply:
will any of them test the case we're dealing with?

Reply:
I'm adding a test into these files.